.github/workflows/release.yml

```diff
@@ -32,10 +32,29 @@ jobs:
         with:
           name: windows-artifacts
           path: /tmp/artifacts
-      - name: Set DRY_RUN_OPTION env variable
-        if: github.event.inputs.dry-run
-        run: echo "DRY_RUN_OPTION=--dry-run" >> $GITHUB_ENV
       - name: Deploy
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: dotnet run --project tools-shared/FakeItEasy.Deploy -- $DRY_RUN_OPTION -r "FakeItEasy/FakeItEasy.Analyzers" -t "${{ github.ref_name }}" -a "/tmp/artifacts/output"
+        run: |
+          if [[ "${{ github.event.inputs.dry-run }}" == "true" ]]; then
+            DRY_RUN_OPTION=--dry-run
+          fi
+          dotnet run --project tools-shared/FakeItEasy.Deploy -- $DRY_RUN_OPTION -r "${{ github.repository }}" -t "${{ github.ref_name }}" -a "/tmp/artifacts/output"
+
+  deploy-docs:
+    needs: deploy
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Trigger docs generation
+        run: |
+          publishMode=auto-merge
+          if [[ "${{ github.event.inputs.dry-run }}" == "true" ]]; then
+            publishMode=none
+          fi
+          gh workflow run generate-docs.yml \
+            -R FakeItEasy/fakeiteasy.github.io \
+            -f version=${{ github.ref_name }} \
+            -f source-repo=${{ github.repository }} \
+            -f source-ref=${{ github.ref_name }} \
+            -f output-prefix=docs/analyzers \
+            -f publish-mode=$publishMode

```

Comment:
Looks good. I'll make a very minor comment. My inclination would be to bunch the stuff that varies together, if at all possible. Maybe hoist the output-prefix up near the -R. That way all the boilerplate is easier to gloss over…

(varies in a different repo, I guess I mean). 

This is the MOST minor of comments, and I wish I hadn't made it. It's not worth doing, considering that it will trigger more testing.